js/app.js

```diff
@@ -3,278 +3,245 @@
-  Prism.highlightAll()
+for (const heading of $headings) {
+  const rect = heading.getBoundingClientRect();
+  const win = heading.ownerDocument.defaultView;
 
-  // menu bar
-
-  var prev;
-  var n = 0;
-
-  var headings = $('h2, h3').map(function(i, el){
-    return {
-      top: $(el).offset().top - 200,
-      id: el.id
-    }
+  headings.push({
+	top: rect.top + win.pageYOffset - 200,
+	id: heading.id,
   });
+}
 
-  function closest() {
-    var h;
-    var top = $(window).scrollTop();
-    var i = headings.length;
-    while (i--) {
-      h = headings[i];
-      if (top >= h.top) return h;
-    }
-  }
-
-  var currentApiPrefix;
-  var parentMenuSelector;
-  var lastApiPrefix;
-
-  $(window).bind('load resize', function() {
-
-    $('#menu').css('height', ($(this).height() - 150) + 'px');
+function closest() {
+  let h;
+  const top = document.scrollingElement.scrollTop;
 
-  });
+  let i = headings.length;
 
-  $(document).scroll(function() {
+  while (i--) {
+	h = headings[i];
 
-    var h = closest();
-    if (!h) return;
+	if (top >= h.top) return h;
+  }
+}
 
+window.addEventListener("scroll", () => {
+  const h = closest();
+  if (!h) return;
 
-    if (window.location.pathname == '/3x/api.html') {
+  currentApiPrefix = h.id.split(".")[0];
+  parentMenuSelector = document.querySelector(`#${currentApiPrefix}-menu`);
+  parentMenuSelector.classList.add("active");
 
-      if (prev) {
-      prev.removeClass('active');
-      prev.parent().parent().removeClass('active');
-      }
-      var a = $('a[href="#' + h.id + '"]');
-      a.addClass('active');
-      a.parent().parent().addClass('active');
-      prev = a;
+  if (lastApiPrefix && lastApiPrefix !== currentApiPrefix) {
+	document.querySelector(`#${lastApiPrefix}-menu`).classList.remove("active");
+  }
 
-    }
+  document.querySelectorAll("#menu li a").forEach(el => el.classList.remove("active"));
 
-    else {
+  document.querySelector(`a[href="#${h.id}"]`).classList.add("active");
 
-      currentApiPrefix = h.id.split('.')[0];
-      parentMenuSelector = '#'+ currentApiPrefix + '-menu';
+  lastApiPrefix = currentApiPrefix.split(".")[0];
+});
 
-      $(parentMenuSelector).addClass('active');
+//  ------------------- cookies -----------------------------
 
-      if (lastApiPrefix && (lastApiPrefix != currentApiPrefix)) {
-        $('#'+ lastApiPrefix + '-menu').removeClass('active');
-      }
+function createCookie(name, value, days) {
+  let expires;
 
-      $('#menu li a').removeClass('active');
+  if (days) {
+	const date = new Date();
 
-      var a = $('a[href="#' + h.id + '"]');
-      a.addClass('active');
+	date.setTime(date.getTime() + days * 24 * 60 * 60 * 1000);
 
-      lastApiPrefix = currentApiPrefix.split('.')[0];
+	expires = `; expires=${date.toGMTString()}`;
+  } else {
+	expires = "";
+  }
 
-    }
+  document.cookie = `${encodeURIComponent(name)}=${encodeURIComponent(value)}${expires}; path=/`;
+}
 
-  })
-  $('#tags-side-menu li').on('click', function() {
```

Comment:
Not sure if this has any function for the blog or if it was just something that was forgotten to be removed along with the blog's menu tags.

Reply:
@chrisdel101 may know